src/switchroot/ostree-mount-util.h

```diff
@@ -114,6 +120,46 @@ read_proc_cmdline_key (const char *key)
   return ret;
 }
 
+static inline char *
+get_aboot_root_slot (void)
+{
+  char __attribute__ ((cleanup (free_char))) *slot_suffix
```

Comment:
I think if we start using cleanup attribute in prepare-root we should add a define for this.

Reply:
It's actually recommended in the coding style "Use GCC `__attribute__((cleanup))` wherever possible. ", not sure how many read it, no extra dependencies and is nicer than manual cleanup.

I'll add some sort of define, the real string is long winded.

Reply:
Yeah, this is used in the rest of ostree but called `g_autofree`. So, maybe just #define it to `autofree`.